core_v2/sources/v2_price_model.move

```diff
@@ -0,0 +1,121 @@
+module aptos_names_v2::v2_price_model {
+    use aptos_names_v2::v2_config;
+    use aptos_std::math64;
+    use std::error;
+
+
+    /// The domain length is too short- currently the minimum is 2 characters
+    const EDOMAIN_TOO_SHORT: u64 = 1;
+    const SECONDS_PER_YEAR: u64 = 60 * 60 * 24 * 365;
+
+    /// The longer the name is registered for, the more expensive it is per year.
+    /// The curve is exponential, with every year costing more than the previous
+    fun scale_price_for_years(price: u64, years: u8): u64 {
```

Comment:
again, why did we keep this code, we are only support 1 year buy outs for a fixed rate... can we please remove this.